etl/steps/data/garden/excess_mortality/latest/excess_mortality_economist/excess_mortality_economist.meta.yml

```diff
@@ -0,0 +1,133 @@
+dataset:
+  title: Excess Mortality Estimates – The Economist (2023)
```

Comment:
I would change the name to "Excess Mortality (The Economist, 2023)", to be more consistent with the other dataset ["Excess Mortality (various sources)"](https://owid.cloud/admin/datasets)